data/curated/border-patrol/meta.yaml

```diff
@@ -0,0 +1,19 @@
+title: U.S. Customs and Border Protection (CBP) encounter data
+article:
+  title: U.S. Border Patrol Encounters
+  url: https://posit.cloud/content/9130042
+data_source:
+  title: U.S. Customs and Border Protection
+  url: https://www.cbp.gov/
+images:
+# Please include at least one image, and up to three images
+- file: inadmissible-aliens-plot.png
+  alt: >
+    A bar plot of monthly inadmissible CHNV (citizens of Cuba, Haiti, Nicaragua, and Venezuela) aliens encountered by U.S. Border Patrol agents from 2020 to 2024.
+credit:
+# We want to thank you for curating this dataset! If you do not want a 
+# particular type of credit, please delete the related line.
+  post: Tony Galván
```

Comment:
I *think* I use parts of this to auto-credit you once I use a command to "assign" this to a week, so we might have an edit to do in the post to make sure we don't double up. No worries, but let's remember to check that!

I don't have it in the template yet, but are you on bluesky yet? If so, add a `bluesky:` line. If not... [join bluesky](https://bsky.app/)! It's where all the fun #RStats conversations are happening! And then add a `bluseky:` line 😁